PyInstaller/lib/modulegraph/modulegraph.py

```diff
@@ -2719,8 +2719,14 @@ def get_operation_arg_name():
             list of names is the unsigned short (i.e., 2-byte integer) argument
             passed to the current bytecode operation.
             """
-            co_names_index = get_operation_arg()
-            return module_code_object.co_names[co_names_index]
+            try:
+                co_names_index = get_operation_arg()
+                return module_code_object.co_names[co_names_index]
+            except IndexError as e:
+                print('co_names_index: {}'.format(co_names_index))
+                print('co_names: {}'.format(module_code_object.co_names))
+                print(repr(self))
```

Comment:
I suggest printing a meaningful message here about what happed where and what the info should tell. Experience teaches that users running into issues here will pop up at the issue-tracker with bad bug reports.

If this code meant only for development, it should not be commited.